demo/src/main/scala/io/finch/demo/Main.scala

```diff
@@ -27,7 +27,7 @@ import _root_.argonaut._
 import com.twitter.finagle.httpx.{Request, Response}
 import com.twitter.finagle.{Filter, Httpx, Service, SimpleFilter}
 import com.twitter.util.{Await, Future}
-import io.finch.{Endpoint => _, _}
+import io.finch._
```

Comment:
Sorry. I was just trying to understand a reason why you decided to not exclude `Endpoint`. Basically, you're right: we don't use either of them so we don't need to exclude.